static/templates/draft_table_body.handlebars

```diff
@@ -8,6 +8,9 @@
                 </div>
             </div>
             <div class="drafts-list">
+                <div class="removed-drafts">
+                    {{t 'Drafts older than' draft_lifetime t 'days are automatically removed'}}
```

Comment:
Woah, I'm not sure to this syntax, though I see why it sorta works.  This isn't valid i18n -- won't work for languages with a different word order than English.  See how we handle similar things elsewhere.